samples/boards/stm32/power_mgmt/standby_shutdown/sample.yaml

```diff
@@ -0,0 +1,19 @@
+sample:
+  name: STM32 GPIO Power Management
+tests:
+  sample.boards.stm32.power_mgmt.standby_shutdown:
+    platform_allow: nucleo_l476rg
```

Comment:
It would be nice to get this working on disco_l475_iot1 which is the L4 target used on bench.
Can we instead filter on CONFIG_SOC_SERIES_STM32L4X ?
(I guess the only condition is to know if config_wakeup_features is also valid for most other L4 targets)